components/bio-formats-tools/test/loci/formats/tools/ImageConverterTest.java

```diff
@@ -124,6 +133,25 @@ public void testDefault(String suffix) throws FormatException, IOException {
     }
   }
 
+  @Test(dataProvider = "options")
+  public void testOptions(String options) throws FormatException, IOException {
+    outFile = File.createTempFile("test", ".ome.tiff");
+    outFile.delete();
```

Comment:
Hmmm. If you're having to do a lot of them, don't know if you could have test setup create a unique *directory* then just create all the filenames in that by incrementing a counter or something. (Guava gives you `createTempDir`.)